OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -37,6 +37,9 @@ public class MobileInfo : PausableConditionalTraitInfo, IMoveInfo, IPositionable
 
 		public readonly int Speed = 1;
 
+		[Desc("If set to true, this unit will always turn in place instead of following a curved trajectory (like infantry).")]
+		public readonly bool AlwaysTurn = false;
```

Comment:
`AlwaysTurn` sounds a bit odd to me. Wouldn't `TurnInPlace` be more descriptive?